vendor/vendor.json

```diff
@@ -461,39 +461,40 @@
-		{"path":"golang.org/x/text/transform","checksumSHA1":"ziMb9+ANGRJSSIuxYdRbA+cDRBQ=","revision":"b7ef84aaf62aa3e70962625c80a571ae7c17cb40","revisionTime":"2018-02-22T12:58:23Z"},
-		{"path":"golang.org/x/text/unicode/bidi","checksumSHA1":"iB6/RoQIzBaZxVi+t7tzbkwZTlo=","revision":"88f656faf3f37f690df1a32515b479415e1a6769","revisionTime":"2017-10-26T07:52:28Z"},
-		{"path":"golang.org/x/text/unicode/cldr","checksumSHA1":"jer43nmInsJhYFfpl39FpEYyoC4=","revision":"b7ef84aaf62aa3e70962625c80a571ae7c17cb40","revisionTime":"2018-02-22T12:58:23Z"},
-		{"path":"golang.org/x/text/unicode/norm","checksumSHA1":"BCNYmf4Ek93G4lk5x3ucNi/lTwA=","revision":"b7ef84aaf62aa3e70962625c80a571ae7c17cb40","revisionTime":"2018-02-22T12:58:23Z"},
-		{"path":"golang.org/x/text/width","checksumSHA1":"zuieOGXKG9vXxNhEWv3H0X8RSXM=","revision":"b7ef84aaf62aa3e70962625c80a571ae7c17cb40","revisionTime":"2018-02-22T12:58:23Z"},
-		{"path":"golang.org/x/time/rate","checksumSHA1":"h/06ikMECfJoTkWj2e1nJ30aDDg=","revision":"a4bde12657593d5e90d0533a3e4fd95e635124cb","revisionTime":"2016-02-02T18:34:45Z"},
+		{"path":"golang.org/x/oauth2","checksumSHA1":"g7j78Uo/JfzkwZqReaVsEhA5G3E=","revision":"bf48bf16ab8d622ce64ec6ce98d2c98f916b6303","revisionTime":"2020-01-07T16:11:21Z"},
+		{"path":"golang.org/x/oauth2/google","checksumSHA1":"w5SmJwdVFlGE1KO4/AQjY+jvdsI=","revision":"bf48bf16ab8d622ce64ec6ce98d2c98f916b6303","revisionTime":"2020-01-07T16:11:21Z"},
+		{"path":"golang.org/x/oauth2/internal","checksumSHA1":"rnUL+5Yzlt+Ky2dlB61VqfhBOb8=","revision":"bf48bf16ab8d622ce64ec6ce98d2c98f916b6303","revisionTime":"2020-01-07T16:11:21Z"},
+		{"path":"golang.org/x/oauth2/jws","checksumSHA1":"huVltYnXdRFDJLgp/ZP9IALzG7g=","revision":"bf48bf16ab8d622ce64ec6ce98d2c98f916b6303","revisionTime":"2020-01-07T16:11:21Z"},
+		{"path":"golang.org/x/oauth2/jwt","checksumSHA1":"HGS6ig1GfcE2CBHBsi965ZVn9Xw=","revision":"bf48bf16ab8d622ce64ec6ce98d2c98f916b6303","revisionTime":"2020-01-07T16:11:21Z"},
+		{"path":"golang.org/x/sync/errgroup","checksumSHA1":"iEK5hCRfrkdc1JOJsaiWuymHmeQ=","revision":"43a5402ce75a95522677f77c619865d66b8c57ab","revisionTime":"2020-03-17T01:21:48Z"},
+		{"path":"golang.org/x/sys/cpu","checksumSHA1":"/b8Fw69AsJ3HgYjSxafhU5VrgGE=","revision":"59c9f1ba88faf592b225274f69c5ef1e4ebacf82","revisionTime":"2020-04-08T23:00:25Z"},
+		{"path":"golang.org/x/sys/unix","checksumSHA1":"2hkstPGuBsXydbG8qaxUjwttfpE=","revision":"59c9f1ba88faf592b225274f69c5ef1e4ebacf82","revisionTime":"2020-04-08T23:00:25Z"},
+		{"path":"golang.org/x/sys/windows","checksumSHA1":"ZYmvJ1qqXG5KHRKVs1tRsRMxj/Q=","revision":"59c9f1ba88faf592b225274f69c5ef1e4ebacf82","revisionTime":"2020-04-08T23:00:25Z"},
+		{"path":"golang.org/x/sys/windows/svc","checksumSHA1":"aeOJe5xLqyC3gN07YRyEC4pI2oo=","revision":"59c9f1ba88faf592b225274f69c5ef1e4ebacf82","revisionTime":"2020-04-08T23:00:25Z"},
```

Comment:
I don't see any files changed for these packages, so I assume their shas just changed due to changes in subpackages we don't use?

Reply:
Wow - good spotting!  Almost but more of sibling packages in this case.  `golang.org/x/sys/unix` changed but `golang.org/x/sys` is a single repo, so the sha changed for all including `cpu` and `windows`.